cfme/tests/infrastructure/test_config_management.py

```diff
@@ -151,3 +169,27 @@ def test_ansible_tower_job_templates_tag(request, config_manager, tag):
 # def test_config_system_reprovision(config_system):
 #    # TODO specify machine per stream in yamls or use mutex (by tagging/renaming)
 #    pass
+
+
+@pytest.mark.tier(3)
+@pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type != "Ansible Tower")
+def test_ansible_tower_service_dialog_creation_from_template(request, config_manager,
+        ansible_tower_dialog_through_ui):
+    """
+    Polarion:
+        assignee: nachandr
+        initialEstimate: 1/4h
+        casecomponent: Ansible
+        caseimportance: high
+
+    """
+    try:
+        job_template = config_manager.appliance.collections.ansible_tower_job_templates.all()[0]
+    except IndexError:
+        pytest.skip("No job template was found")
```

Comment:
It look like `setup` means job_template required for this test... 
optional: you can move this as fixture.